sphinxlint/__main__.py

```diff
@@ -58,6 +58,16 @@ def __call__(self, parser, namespace, values, option_string=None):
                     ) from None
             setattr(namespace, self.dest, sort_fields)
 
+    class StoreNumJobsAction(argparse.Action):
+        def __call__(self, parser, namespace, values, option_string=None):
+            setattr(namespace, self.dest, self._job_count(values))
+
+        @staticmethod
+        def _job_count(values):
```

Comment:
There's no public API here, we can remove the underscore to remove an IDE squiggle. Also it matches the existing `SortField.as_supported_options`:
```suggestion
        def job_count(values):
```

Reply:
Sure, that's another symptom of working in other projects where either tools or people will unequivocally complain if there's a public method without a docstring.